### PR TITLE
Cleanup arm32

### PR DESCRIPTION
**Pull request by gkhanna79**
Now that the open build produces the packages, we do not need to depend upon this external dependency.

@weshaggard @dagood PTAL

@dagood Once this PR is merged, we will need to update Maestro to no longer update the ExternalDependency for CoreCLR master branch.

FYI - @RussKeldorph

### CODE REVIEW

**Comment by gkhanna79** [comment]
FYI @ianhays

File: dependencies.props [hunk 1]
```diff
@@ -9,7 +9,6 @@
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
     <CoreFxExpectedPrerelease>beta-24615-03</CoreFxExpectedPrerelease>
-    <ExternalExpectedPrerelease>beta-24616-00</ExternalExpectedPrerelease>
   </PropertyGroup>
 
   <!-- Full package version strings that are used in other parts of the build. -->
```
Comment (weshaggard) [inline dependencies.props]: Can't you delete `<RemoteDependencyBuildInfo Include="External">` below as well?

Comment (dagood) [inline dependencies.props]: And with that gone, `ExternalCurrentRef` can go too.

**Comment by gkhanna79** [comment]
@dagood @weshaggard PTAL at the second commit where I have removed the other two references both of you pointed out.

**Comment by dagood** [comment]
Thanks, LGTM. Tried out a verify and upgrade locally and they still work.

**Pull request merged**